src/Lib/FileSystem/Image.php

```diff
@@ -29,6 +29,7 @@ public function __construct(string $path)
     {
         parent::__construct($path);
         $this->image = new Imagick($this->getPath());
+        $this->image->setCompressionQuality(70);
```

Comment:
This makes any image instance compressed. Move it to separate method. Also this do nothing, you should write to image https://www.php.net/manual/en/imagick.writeimage.php